CHANGELOG.md

```diff
@@ -1,4 +1,46 @@
 # Gerbil Release Notes
+
+## Gerbil v0.18.1, NimzoLarsen
```

Comment:
Add SSL support for postgres, thats an important feature.

Reply:
I did mention that already. You want it on a bullet list versus a paragraph?